<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>s21_credit</class>
 <widget class="QMainWindow" name="s21_credit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>465</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>440</width>
    <height>465</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>440</width>
    <height>465</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calculator (by kossadda)</string>
  </property>
  <property name="windowIcon">
   <iconset theme="accessories-calculator"/>
  </property>
  <property name="windowOpacity">
   <double>0.975000000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>440</width>
     <height>465</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>440</width>
     <height>465</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(30, 27, 6);</string>
   </property>
   <widget class="QPushButton" name="calculate">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>405</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>–†–∞—Å—Å—á–∏—Ç–∞—Ç—å</string>
    </property>
   </widget>
   <widget class="QComboBox" name="creditBox">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>9</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>200</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox { background-color: rgb(30, 27, 6); border: 15px; text-align: center; } QComboBox::drop-down { border: none; }</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="calc_type">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>101</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>135</width>
      <height>400</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(30, 27, 6);</string>
    </property>
    <property name="text">
     <string>–í–∞—Ä–∏–∞–Ω—Ç —Ä–∞—Å—á–µ—Ç–∞</string>
    </property>
   </widget>
   <widget class="QLabel" name="info_label_1">
    <property name="geometry">
     <rect>
      <x>149</x>
      <y>106</y>
      <width>17</width>
      <height>17</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;–ö–∞–ª—å–∫—É–ª—è—Ç–æ—Ä –ø—Ä–æ–∏–∑–≤–æ–¥–∏—Ç 3 –≤–∞—Ä–∏–∞–Ω—Ç–∞ —Ä–∞—Å—á–µ—Ç–æ–≤:&lt;/p&gt;&lt;p&gt;1. –†–∞—Å—á–µ—Ç –µ–∂–µ–º–µ—Å—è—á–Ω–æ–≥–æ –ø–ª–∞—Ç–µ–∂–∞ –ø–æ —Å—É–º–º–µ –∏ —Å—Ä–æ–∫—É –∫—Ä–µ–¥–∏—Ç–∞.&lt;/p&gt;&lt;p&gt;2. –†–∞—Å—á–µ—Ç –µ–∂–µ–º–µ—Å—è—á–Ω–æ–≥–æ –∫—Ä–µ–¥–∏—Ç–∞ –ø–æ —Å—É–º–º–µ –∏ –∑–∞—Ä–∞–Ω–µ–µ –∑–∞–¥–∞–Ω–Ω–æ–º—É –µ–∂–µ–º–µ—Å—è—á–Ω–æ–º—É –ø–ª–∞—Ç–µ–∂—É.&lt;/p&gt;&lt;p&gt;3. –†–∞—Å—á–µ—Ç –≤–æ–∑–º–æ–∂–Ω–æ–π —Å—É–º–º—ã –∫—Ä–µ–¥–∏—Ç–∞ –ø—Ä–∏ —Ñ–∏–∫—Å–∏—Ä–æ–≤–∞–Ω–Ω–æ–º —Å—Ä–æ–∫–µ –∏ –ø–ª–∞—Ç–µ–∂–µ.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolTip {
	color: white;
	background-color: black;
	border: 1px solid white;
	border-radius: 15px;
}
</string>
    </property>
    <property name="text">
     <string>üõà</string>
    </property>
   </widget>
   <widget class="QComboBox" name="calc_type_box">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>95</y>
      <width>237</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="sum_line">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>153</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="sum">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>159</y>
      <width>115</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>125</width>
      <height>400</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(30, 27, 6);</string>
    </property>
    <property name="text">
     <string>–°—É–º–º–∞ –∫—Ä–µ–¥–∏—Ç–∞</string>
    </property>
   </widget>
   <widget class="QComboBox" name="sum_box">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>153</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="time">
    <property name="geometry">
     <rect>
      <x>62</x>
      <y>219</y>
      <width>106</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>110</width>
      <height>400</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(30, 27, 6);</string>
    </property>
    <property name="text">
     <string>–°—Ä–æ–∫ –∫—Ä–µ–¥–∏—Ç–∞</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="time_line">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>211</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QComboBox" name="time_box">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>211</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="percent">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>277</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>400</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(30, 27, 6);</string>
    </property>
    <property name="text">
     <string>–ü—Ä–æ—Ü–µ–Ω—Ç–Ω–∞—è —Å—Ç–∞–≤–∫–∞</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="percent_line">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>269</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="info_label_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>275</y>
      <width>15</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QLabel" name="payment_1">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>330</y>
      <width>135</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>220</width>
      <height>400</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(30, 27, 6);</string>
    </property>
    <property name="text">
     <string>–¢–∏–ø –µ–∂–µ–º–µ—Å—è—á–Ω—ã—Ö</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="payment_type_1">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>325</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string>–ê–Ω–Ω—É–∏—Ç–µ—Ç–Ω—ã–µ</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="payment_type_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>360</y>
      <width>172</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string>–î–∏—Ñ—Ñ–µ—Ä–µ–Ω—Ü–∏—Ä–æ–≤–∞–Ω–Ω—ã–µ</string>
    </property>
   </widget>
   <widget class="QLabel" name="payment_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>360</y>
      <width>72</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>220</width>
      <height>400</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(30, 27, 6);</string>
    </property>
    <property name="text">
     <string>–ø–ª–∞—Ç–µ–∂–µ–π</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>61</y>
      <width>428</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="info_label_3">
    <property name="geometry">
     <rect>
      <x>303</x>
      <y>329</y>
      <width>17</width>
      <height>17</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;–ê–Ω–Ω—É–∏—Ç–µ—Ç–Ω—ã–π –ø–ª–∞—Ç–µ–∂ - –≤–∞—Ä–∏–∞–Ω—Ç –µ–∂–µ–º–µ—Å—è—á–Ω–æ–≥–æ –ø–ª–∞—Ç–µ–∂–∞ –ø–æ –∫—Ä–µ–¥–∏—Ç—É, –∫–æ–≥–¥–∞ —Ä–∞–∑–º–µ—Ä –µ–∂–µ–º–µ—Å—è—á–Ω–æ–≥–æ –ø–ª–∞—Ç–µ–∂–∞ –æ—Å—Ç–∞—ë—Ç—Å—è –ø–æ—Å—Ç–æ—è–Ω–Ω—ã–º –Ω–∞ –≤—Å—ë–º –ø–µ—Ä–∏–æ–¥–µ –∫—Ä–µ–¥–∏—Ç–æ–≤–∞–Ω–∏—è.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolTip {
	color: white;
	background-color: black;
	border: 1px solid white;
	border-radius: 15px;
}
</string>
    </property>
    <property name="text">
     <string>üõà</string>
    </property>
   </widget>
   <widget class="QLabel" name="info_label_4">
    <property name="geometry">
     <rect>
      <x>366</x>
      <y>365</y>
      <width>17</width>
      <height>17</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;–î–∏—Ñ—Ñ–µ—Ä–µ–Ω—Ü–∏—Ä–æ–≤–∞–Ω–Ω—ã–π –ø–ª–∞—Ç–µ–∂ - –≤–∞—Ä–∏–∞–Ω—Ç –µ–∂–µ–º–µ—Å—è—á–Ω–æ–≥–æ –ø–ª–∞—Ç–µ–∂–∞ –ø–æ –∫—Ä–µ–¥–∏—Ç—É, –∫–æ–≥–¥–∞ —Ä–∞–∑–º–µ—Ä –µ–∂–µ–º–µ—Å—è—á–Ω–æ–≥–æ –ø–ª–∞—Ç–µ–∂–∞ –ø–æ –ø–æ–≥–∞—à–µ–Ω–∏—é –∫—Ä–µ–¥–∏—Ç–∞ –ø–æ—Å—Ç–µ–ø–µ–Ω–Ω–æ —É–º–µ–Ω—å—à–∞–µ—Ç—Å—è –∫ –∫–æ–Ω—Ü—É –ø–µ—Ä–∏–æ–¥–∞ –∫—Ä–µ–¥–∏—Ç–æ–≤–∞–Ω–∏—è.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolTip {
	color: white;
	background-color: black;
	border: 1px solid white;
	border-radius: 15px;
}
</string>
    </property>
    <property name="text">
     <string>üõà</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
